README.md

```diff
@@ -0,0 +1,529 @@
+# Memo Bank's growth guide
+
+Memo Bank's growth guide is a financing guide aimed at established companies that are still growing fast (“scaling“ companies). The guide is maintained by [Memo Bank](https://memo.bank/en), a French bank built by entrepreneurs, for entrepreneurs who run small and medium businesses.
+
+This repository contains the code used to publish our guide at:[guide.memo.bank](https://guide.memo.bank). We use [Jekyll](https://jekyllrb.com) with the [Just the Docs](https://just-the-docs.github.io/just-the-docs/) theme. This site is built and served by [GitHub Page](https://pages.github.com).
```

Comment:
There is a space missing:

```diff
+ This repository contains the code used to publish our guide at: [guide.memo.bank](https://guide.memo.bank).
- This repository contains the code used to publish our guide at:[guide.memo.bank](https://guide.memo.bank). 
```

Reply:
Thanks @dirtyhenry. I'll fix it in a forthcoming commit. 👍